cpp/src/parquet/arrow/schema.h

```diff
@@ -91,7 +91,6 @@ struct PARQUET_EXPORT SchemaField {
   std::shared_ptr<::arrow::Field> field;
   // If field has an extension type, an equivalent field with the storage type,
```

Comment:
remove the docs also?

Reply:
Alos, just want to double check that we can always get the raw underlying type from the extension types?  One place where this code could cause some issues is if a leaf column is filtered out but the extension type requires all columns. (this doesn't necessarily apply to this code review as i think this was potentially and already existng bug).

Reply:
Ah, yes, will remove, sorry.
You may be right about the issue, though I'm not sure how one would filter out a column from an extension's storage type.

Reply:
To provide more context https://github.com/apache/arrow/blob/master/cpp/src/parquet/arrow/reader.cc#L791 GetReader can return a nullptr for a reader, and for struct fields the Types can change if a specific one is filtered https://github.com/apache/arrow/blob/master/cpp/src/parquet/arrow/reader.cc#L840

Reply:
It would be nice if you could open a JIRA with a reproducer, or at least guidance as to how this can be reproduced.